### PR TITLE
update ex and sol sheet 9

### PR DESCRIPTION
**Pull request by JuliaHerbinger**
updated ex sheet 9 (Rnw and pdf)

### CODE REVIEW

File: exercises/regularization/ex_rnw/ex_lasso_subdifferential.Rnw [hunk 1]
```diff
@@ -17,16 +17,16 @@ Optimization routines for the Lasso use coordinate gradient descent, but instead
           c(expression(x^2), expression(2*x^2), expression(paste("max(0,", 0.8*x, ")")), "|x|"))
 @
 
-  \item A subdifferential of $f$ is a set of values $\breve{\nabla}_x f$ defined as $$\breve{\nabla}_x f = \{ g: f(x) \geq f(x_0) + g \cdot (x-x_0) \, \forall x \in \mathcal{I} \}.$$ Every scalar value $g \in \breve{\nabla}_x$ is said to be a subgradient of $f$.  Does a subdifferential have any parallels to the previous question? How can we interpret $g$?
+  \item A subdifferential of $f$ is a set of values $\breve{\nabla}_{x_0} f$ defined as $$\breve{\nabla}_{x_0} f = \{ g: f(x) \geq f(x_0) + g \cdot (x-x_0) \, \forall x \in \mathcal{I} \}.$$ Every scalar value $g \in \breve{\nabla}_{x_0}$ is said to be a subgradient of $f$.  Does a subdifferential have any parallels to the previous question? How can we interpret $g$?
```
Comment (ludwig-max) [inline exercises/regularization/ex_rnw/ex_lasso_subdifferential.Rnw:20]: "is said to be a subgradient of f *at x_0*"

File: exercises/regularization/ex_rnw/sol_lasso_subdifferential.Rnw [hunk 2]
```diff
@@ -16,15 +16,15 @@ plot(xx, yy, type = "l", xlab = "x", ylab = "f(x)", ylim=c(-4,10), col ="red", l
 for(x0 in seq(-2,2,by=0.5))
```
Comment (ludwig-max) [inline exercises/regularization/ex_rnw/sol_lasso_subdifferential.Rnw:16]: oben in Zeile 5 löschen: "and for $x=0$ exactly the function $2x^2$ mirrored at the $f(x) = 0$ axis."

**Review by ludwig-max — CHANGES_REQUESTED** [review]
2 Kleinigkeiten, danke Dir!

File: exercises/regularization/ex_rnw/ex_lasso_subdifferential.Rnw (hunk 1 above)
Comment (JuliaHerbinger) [reply exercises/regularization/ex_rnw/ex_lasso_subdifferential.Rnw]: updated with last commit fe8dc3a11d71afcc3d170f2e580b71ffba1c691b

File: exercises/regularization/ex_rnw/sol_lasso_subdifferential.Rnw (hunk 2 above)
Comment (JuliaHerbinger) [reply exercises/regularization/ex_rnw/sol_lasso_subdifferential.Rnw:16]: updated with last commit fe8dc3a11d71afcc3d170f2e580b71ffba1c691b

**Review by ludwig-max — APPROVED** [review]

**Pull request merged**